### PR TITLE
Add a script to run reftests and spawn reftest analyzer.

### LINKED ISSUE #1863: Add a wrench option to auto-spawn reftest analyzer on failed reftest run.

### PR DESCRIPTION
**Pull request by glennw**
This probably only works on Linux, and will need some changes
to support running on Mac and Windows.

Fixes #1863.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/1907)

### CODE REVIEW

**Comment by glennw** [comment]
r? anyone

File: wrench/reftest-debugger.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import subprocess
+import urllib
```
Comment (kvark) [inline wrench/reftest-debugger.py]: > /wrench/reftest-debugger.py:8: F401 'urllib' imported but unused

**Comment by glennw** [comment]
@kvark Oops, fixed.

**Comment by kvark** [comment]
Did you consider moving both this one and `benchmark_server` to `wrench/script` folder? Otherwise r=me

**Comment by glennw** [comment]
Probably a good idea to move those to a separate directory, will do as a follow up.

@bors-servo r=kvark

**Pull request merged**